lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -85,6 +96,7 @@ async function runA11yChecks() {
   // Augment the node objects with outerHTML snippet & custom path string
   axeResults.violations.forEach(augmentAxeNodes);
   axeResults.incomplete.forEach(augmentAxeNodes);
+  axeResults.inapplicable.forEach(augmentAxeNodes);
```

Comment:
why is this needed now? if something's inapplicable, it shouldn't have any nodes at all or it would be applicable...

Reply:
help url change. Didn't want to recreate the looping stuff for just that.